### DISCUSSION TITLE
Github 2FA only on a single account

### DISCUSSION THREAD

**Discussion by 1alexvash-work** [community/community#143250] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there, I have two github accounts

One is older, and this one is newer.
Why was I only asked to set up 2FA authorization only on the newer account?
I think I had more activity on the older account, is there any rule or criteria by which accounts were selected, as it's a bit odd that I'm going to have 2FA on one account but not on the another one

**Comment by thiha-min-thant** [comment]
In general, GitHub may encourage users to enable 2FA based on various criteria, but there isn't a specific rule that limits the prompt to just one account. Some reasons why you might have been prompted to enable 2FA only on your newer account could include:

1. Account Settings: Sometimes, newer accounts or accounts with recent security-related changes might receive a prompt to enable 2FA, while older accounts without recent updates might not.
2. Organization or Team Membership: If the newer account is a member of an organization or team with mandatory 2FA, GitHub will require you to enable it for that account.
3. GitHub's Rolling Prompts: GitHub periodically encourages or mandates 2FA for accounts, but this may happen at different times for different accounts.

You can always enable 2FA manually on your older account if you want consistent security across both accounts. To do this, go to Settings > Password and authentication > Two-factor authentication and follow the setup instructions.

[About mandatory two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-mandatory-two-factor-authentication)
For more information about the 2FA enrollment rollout, see [this blog post](https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13).
